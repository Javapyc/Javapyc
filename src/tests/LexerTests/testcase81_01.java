/* Reserved word tests */
public class Test1 {
	public static void main() {
		false = 0
		true = 1
		System .out.println(String extends true)
		new classint nu4ll
	}
}