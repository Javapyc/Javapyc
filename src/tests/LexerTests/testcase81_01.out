ReservedWord, public
ReservedWord, class
ID, Test1
Delimiter, {
ReservedWord, public
ReservedWord, static
ReservedWord, void
ReservedWord, main
Delimiter, (
Delimiter, )
Delimiter, {
ReservedWord, false
Delimiter, =
Integer, 0
ReservedWord, true
Delimiter, =
Integer, 1
ID, System
Delimiter, .
ID, out
Delimiter, .
ID, println
Delimiter, (
ReservedWord, String
ReservedWord, extends
ReservedWord, true
Delimiter, )
ReservedWord, new
ID, classint
ID, nu4ll
Delimiter, }
Delimiter, }
